scripts/rust-analyze.sh

```diff
@@ -49,6 +51,11 @@ if [ -d "$OBJDIR" ]; then
   if [ -d "$INDEX_ROOT/rustlib" ]; then
     ANALYSIS_DIRS="$ANALYSIS_DIRS $(find $INDEX_ROOT/rustlib -type d -path \*/${PLATFORM_FILTER}analysis)"
   fi
+  # If a sed normalization command was provided, perform the transform over all
+  # json analysis files.
+  if [[ $SED_TRANSFORM_EXPR ]]; then
+    find $ANALYSIS_DIRS -type f -name \*.json | parallel --halt now,fail=1 sed --in-place -e "\"$SED_TRANSFORM_EXPR\"" {}
```

Comment:
I'd prefer `"*.json"` to `\*.json` (i.e. quotes instead of backslash. I'm also not sure why the extra backslashed-quotes are required around `$SED_TRANSFORM_EXPR` here.

Reply:
I'll change the wildcarding as I now push again with the $4 fix.  (I plead incompetence.)

When locally playing with this it seemed necessary.  Here's my noodling with all my cool prompt stuff making it hard to read.
```
 visbrero  …  code  searchfox-play  test                     
 $  SED_TRANSFORM_EXPR="s#builds/worker/workspace/build/src/##g;s#z:/task_[0-9]*/build/src/##gI"
 visbrero  …  code  searchfox-play  test                     
 $  /usr/bin/find $ANALYSIS_DIRS -type f -name \*.json | parallel --halt now,fail=1 sed --in-place -e "$SED_TRANSFORM_EXPR" {}
sed: no input files
/bin/bash: s#z:/task_[0-9]*/build/src/##gI: No such file or directory
parallel: This job failed:
sed --in-place -e s#builds/worker/workspace/build/src/##g;s#z:/task_[0-9]*/build/src/##gI foo/kaboom.json
 visbrero  …  code  searchfox-play  test  0  127                     
 0  127  $  /usr/bin/find $ANALYSIS_DIRS -type f -name \*.json | parallel --halt now,fail=1 sed --in-place -e "\"$SED_TRANSFORM_EXPR\"" {}
 visbrero  …  code  searchfox-play  test                     
```